tests/integration/security/util/reachability/context.go

```diff
@@ -171,69 +113,102 @@ func (rc *Context) Run(testCases []TestCase) {
 			// to work around this, we will temporarily make sure we are always sleeping at least 10s, even if istioctl wait is faster.
 			// This allows us to debug istioctl wait, while still ensuring tests are stable
 			sleep := time.Second*10 - time.Since(t0)
+			if !deployedWithinWait {
+				sleep = time.Second * 30
```

Comment:
is this temporary? same comment about masking a bug

Reply:
Removed. Same as timeout